.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/test.md
```

Comment:
- [ ] Both node_modules and test.md files should be on separate lines so that they won't be read as a directory.